kdc/kerberos5.c

```diff
@@ -1859,6 +1887,20 @@ _kdc_as_rep(kdc_request_t r,
 	goto out;
     }
 
+    /* If the server is not krbtgt, fetch local krbtgt key to sign authdata */
+    if (!is_tgs) {
+
```

Comment:
this blank line can go :)

Reply:
I'll make it go.
I also wonder if we should be more graceful here and not error out but just dismiss authdata, though I'm not sure that's really necessary.